grants/crosschain_defenders/data.md

```diff
@@ -0,0 +1,60 @@
+---
+Grant-Number: "010",
```

Comment:
Crosschain Defenders metadata contains Ramp networks details. Please check and update as you see fit.